presto-main/src/main/java/io/prestosql/sql/planner/SubqueryPlanner.java

```diff
@@ -527,31 +527,37 @@ private PlanNode replaceExpressionsWithSymbols(PlanNode planNode, Map<NodeRef<Ex
     }
 
     private static class ColumnReferencesExtractor
-            extends DefaultExpressionTraversalVisitor<Void, ImmutableSet.Builder<NodeRef<Expression>>>
+            extends DefaultExpressionTraversalVisitor<Void, Void>
```

Comment:
Then object of this class cannot be reused.

Reply:
Yes, and it's not reused (there is no point to).
This is consistent with previous such changes: https://github.com/prestosql/presto/commit/0a971051d8bd703c0259c82d203cbb8aef70fee2